### DISCUSSION TITLE
Pre-install scripts fail during installation on vmware/virtualbox

### DISCUSSION THREAD

**Discussion by cybersecurityhandle** [Security-Onion-Solutions/securityonion#7787] [Unsupported Versions] [answered]
Hi Security Onion, 
When I attempt to install a new instance of Security Onion via the ISO on both VMware and virtualbox. 
It gets stucks on the pre-installation script and denotes: 
![image](https://user-images.githubusercontent.com/95672692/163267559-95880e91-17cb-409f-b498-7f56f698bc72.png)
The only issue I could imagine it being is that I have to select enter when it says I don't have 99GB of space (I'm using dynamic spacing and I do have that allocation, however this is merely for a POC.

**Comment by dougburks** [comment, marked as answer]
What are the specs of your VM?
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs

**Reply by cybersecurityhandle** [reply]
Base Memory: 9975 MB
Processors: 2
Video Memory: 17 MB

IDE primary Device 0: Security Onion Fresh.vdi (normal, 32 GB) 
Storage Dynamically allocated

**Reply by dougburks** [reply]
Please review the minimum specs from my previous reply:
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs
